src/core/lib/iomgr/error.cc

```diff
@@ -727,10 +727,10 @@ static char *finish_kvs(kv_pairs *kvs) {
 }
 
 const char *grpc_error_string(grpc_error *err) {
-  GPR_TIMER_BEGIN("grpc_error_string", 0);
   if (err == GRPC_ERROR_NONE) return no_error_string;
   if (err == GRPC_ERROR_OOM) return oom_error_string;
   if (err == GRPC_ERROR_CANCELLED) return cancelled_error_string;
+  GPR_TIMER_BEGIN("grpc_error_string", 0);
```

Comment:
Now that we have C++... why not move this back to the head of function and use GPR_TIMER_SCOPE (removing the GPR_TIMER_END)